### PR TITLE
Update branch `2.2`.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

**Review by bizley — APPROVED** [review]

**Review by Arhell — APPROVED** [review]

File: framework/BaseYii.php [hunk 1]
```diff
@@ -93,7 +93,7 @@ class BaseYii
      */
     public static function getVersion()
     {
-        return '2.0.49-dev';
+        return '2.0.50-dev';
```
Comment (mtangoo) [inline framework/BaseYii.php:96]: Why is it 2.0.50-dev? Isn't this supposed to be 2.2-dev?
Other than this I have no objection

**Review by mtangoo — APPROVED** [review]

File: framework/CHANGELOG.md [hunk 2]
```diff
@@ -9,21 +9,32 @@ Yii Framework 2 Change Log
 
 
 2.0.49 under development
+2.0.50 under development
```
Comment (rob006) [inline framework/CHANGELOG.md:12]:
```suggestion
2.0.50 under development
```

File: framework/BaseYii.php (hunk 1 above)
Comment (terabytesoftw) [reply framework/BaseYii.php]: done.

**Comment by terabytesoftw** [comment]
We can merge this PR to continue moving forward in the 2.2 branch.

**Pull request merged**